setup.cfg

```diff
@@ -22,7 +22,10 @@ provides =
 [options]
 python_requires = >=3.6.1
 install_requires =
+    matplotlib >=3.1.1, <=3.6.3, !=3.3.0rc1
```

Comment:
For this specific library I wouldn't think that there would be an issue updating, since the use of matplotlib is limited (only `plot_physio()` which is just an optional utility)

However, these limitations where posed from `peakdet`. I'm not sure if there was any specific reasoning or if there were any conflicts with the manual inspection GUI, maybe @smoia has some info on that

Reply:
I spent some time on this because I have the same error trying to import matplotlib.pyplot in my virtual environment in the terminal (version 3.6.3). However, I added a quick test calling plot_physio() and explicitly importing matplotlib.pyplot which passed when I ran it in pytest!

`def test_import():
    import matplotlib.pyplot as plt
    import matplotlib as matplotlib_pkg
    var = matplotlib_pkg.__version__
    assert var == "3.6.3"
    phys = Physio(DATA, fs=1000)
    ax_phys = phys.plot_physio()
    assert plt.gcf().number == 1`

So, I'm not too sure what is going on here. If anyone has any insight I'd appreciate it, otherwise I guess we can proceed while keeping an eye out for any future version issues.

Reply:
Side note: I would be interested to discuss how we add robust test coverage for plots! 
@maestroque do you have any experience with this?

Reply:
I'm not sure what kind of unit tests for plots you have in mind. For the time being the plotting function here does not have something convoluted (as in peakdet) other than just a plt.plot() call.

Also, again, I don't think there would be an issue updating the matplotlib module, these version constraints were just posed from peakdet. If there is any insight for this then ok, otherwise we can also proceed with 3.9 I don't think there would be an issue. Otherwise if there is indeed a reason for the version constraints in peakdet, we should keep them here because most probably someone would use a common environment for both modules `physutils` and `peakdet`